src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,50 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("size() 테스트")
+    void sizeTest(){
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    @DisplayName("ValueSource 어노테이션을 활용하여 컬렉션이 값을 포함하는지 테스트")
+    void containsValue_WithValueSourceTest(int input){
+        assertThat(numbers).contains(input);
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"1:true","2:true","3:true","4:false","5:false"}, delimiter = ':')
+    @DisplayName("CsvSource 어노테이션을 활용하여 컬렉션에 값 포함 여부 테스트")
+    void containsValue_ShouldReturnBooleanValueWithCsvSourceTest(String intInput, String booleanInput){
```

Comment:
```suggestion
    void containsValue_ShouldReturnBooleanValueWithCsvSourceTest(int number, boolean expected){
```
CsvSource 사용 시 타입을 자동으로 추론하기 때문에 별도 파싱 없이 바로 int, boolean 형으로 사용하셔도 됩니다~!

Reply:
오 타입추론이 되는군요!!! 알려주셔서 감사합니다^^